### DISCUSSION TITLE
各位怎么看待啊b纳塔音乐这个播放量和投币量？

### DISCUSSION THREAD

**Discussion by cabagebird** [share121/inter-knot#3056] [其他]
叔叔/阿姨到底为了什么？
<img src="https://img-static.mihoyo.com/communityweb/upload/366bd6df27c1291e86c699ddc25bc816.png" alt="三月七_无语" width="75" >
感谢Dave-12138提供的表情包
![Snipaste_2024-08-19_15-18-31](https://github.com/user-attachments/assets/1119b255-2ae6-47db-98d9-258dd7547cb4)

**Comment by CJHXAMSTLP** [comment]
米和B之间矛盾越来越大了，看看这个视频就明白了。【「一水」2023年重新评价原神——一把刺穿B站的利剑】 https://www.bilibili.com/video/BV1de4y1c7WH/?share_source=copy_web&vd_source=4c5279effc258362d964677d917764c3

**Comment by Nie7bai** [comment]
蜀黍也是要掐钱的捏<img src="https://upload-bbs.miyoushe.com/upload/2023/03/09/3173bc3e69fcfebecfda741340c9ad22_6841250310756275678.png" alt="开拓者 吃瓜ing" width="75" >

**Comment by LIT-CMD** [comment]
big dan

**Comment by raven-null** [comment]
恰钱惹，节奏乱飞都不管了<img src="https://upload-bbs.miyoushe.com/upload/2023/03/09/3173bc3e69fcfebecfda741340c9ad22_6841250310756275678.png" alt="开拓者 吃瓜ing" width="75" >

**Comment by qyn423520** [comment]
想起之前爱莉的pv，陈叔叔也是要恰烂钱的啊

**Comment by wuyueji** [comment]
躺床上看？（手动阿星吃瓜.jpg)

**Comment by mochengyanliu920** [comment]
b站打压呗

**Comment by showx1ao** [comment]
还能说啥，一边吃瓜一边看咯

**Comment by MTAlexKen** [comment]
![image_22](https://github.com/user-attachments/assets/d43b1f6d-1144-484d-85a6-a276d4d67c88)

**Comment by Alsuhold** [comment]
纯犯贱，挑起冲突好恰饭
